### DISCUSSION TITLE
Plot: Support multiple X/Y series

### DISCUSSION THREAD

**Discussion by ljburtz** [foxglove/community#471] [Q&A] [answered]
Hello, migrating from plotjuggler to foxglove but a currently blocking use case is comparing robot odometry calculated by the robot to a ground truth

for this I need to display an x/y plot -> top down map
(the x axis is the odometry x, the y axis is the odometry y. there is no timestamp indexing as in a regular time series)

this works great in foxglove for a single plot by using "msg path accumulated) as an index (see screenshot, blue line)
![image](https://user-images.githubusercontent.com/26211495/181671710-42dea49e-f3a3-4ef4-a49e-dd72cd94a027.png)

However, I can't find out how to display two (or three) overlapping robot trajectories on the same plot = to compare the robot's odometry estimate with the ground truth

See screenshot of this feature in plotjuggler -> x/y plots make it very intuitive to compare estimated trajectories to ground truth
![image](https://user-images.githubusercontent.com/26211495/181671773-de1c027b-d0e7-4c7b-927f-02aeaf0d9a09.png)
two different trajectory estimates, drifting away from the ground truth as the robot travels

Thanks!

**Comment by ljburtz** [comment]
Maybe in foxglove parlance these types of x/y plots are a separate panel type?
Something closer to the Map panel? But not using a NavSatFix message (longitude / latitude) but simple x and y  coordinates pairs from selectable topic's message field?

basically the Map panel is overkill for many use cases (that don't use longitude/latitude) and the current plot panel is best for timeseries where the plot's x axis is a timestamp, not a coordinate

**Comment by ljburtz** [comment]
this issue also adrresses the feature request in foxglove/community#150 of 1 year ago! :pray:

**Comment by indirectlylit** [comment, marked as answer]
Hello! We recently shipped a feature which allows you to set X values independently for each series in path-based (accumulated and current) XY plots. It's available on web now, and will be in the next desktop release. If you try it and have any feedback, please let us know!
